grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -354,7 +354,7 @@ void onRequestMessage(DeframedMessage message, boolean endOfStream) {
         }
     }
 
-    final void onRequestComplete() {
+    public final void onRequestComplete() {
```

Comment:
```suggestion
    protected final void onRequestComplete() {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!